<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWindowCentralQWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="MainWindowQHBoxLayout" stretch="0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="HeartWaveWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>660</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/overlay/denasButtonless.svg);
background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <widget class="QGraphicsView" name="HeartWaveGraphicsView">
         <property name="geometry">
          <rect>
           <x>3</x>
           <y>1</y>
           <width>601</width>
           <height>661</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="Left">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>440</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Left</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Down">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>510</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Right">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>440</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Right</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Up">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>370</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
        <widget class="QPushButton" name="OnOff">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>600</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>On/Off</string>
         </property>
        </widget>
        <widget class="QPushButton" name="HomeButton">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>300</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Home Button</string>
         </property>
        </widget>
        <widget class="QPushButton" name="BackReturn">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>300</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Back/Return</string>
         </property>
        </widget>
        <widget class="QGraphicsView" name="PowerOffView">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>531</width>
           <height>250</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>250</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>440</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Selector</string>
         </property>
        </widget>
        <widget class="QListWidget" name="mainMenuListView">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>531</width>
           <height>201</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Admin">
         <attribute name="title">
          <string>Admin</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="adminTabQVBoxLayout">
            <item>
             <layout class="QGridLayout" name="adminTabQGridLayout">
              <item row="1" column="1">
               <widget class="QPushButton" name="BatteryRechargeAdminButton">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>Recharge</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="BatteryLevelAdminLabel">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>Battery Level</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QComboBox" name="SensorAttachedAdminBox">
                <property name="maximumSize">
                 <size>
                  <width>82</width>
                  <height>30</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>True</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>False</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QDoubleSpinBox" name="BatteryPercentageAdminBox">
                <property name="maximumSize">
                 <size>
                  <width>82</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="SensorAttachedAdminLabel">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>Sensor Attached</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
